### PR TITLE
Initialize repository

### PR DESCRIPTION
**Pull request by prog893**
Initial code and README to start with.

### CODE REVIEW

**Comment by prog893** [comment]
Testing release tested E2E on real infrastructure

File: README.md [hunk 1]
```diff
@@ -1,32 +1,94 @@
...
+  schedule_expression = "cron(40 1 * * ? *)"
+  is_enabled          = "true"
+
+  target_cluster_arn = "dev"
+
+  task_definition_arn = "${aws_ecs_task_definition.ecs_task_definition.arn}"
+  task_role_arn       = "${module.iam_ecs_tasks.arn}"
+  task_count          = "1"
+
+  subnets         = "subnet-***1,subnet-***2"
```
Comment (toVersus) [inline README.md]: Change variable name from `subnet` to `subnet_ids` and pass a list of subnet ids.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,94 @@
...
+  security_groups = "sg-***1,sg-***2"
```
Comment (toVersus) [inline README.md]: Same as above. Change variable name from `security_groups ` to `security_group_ids` and pass a list of security group ids.

File: README.md [hunk 3]
```diff
@@ -1,32 +1,94 @@
...
+}
+
+module "iam_ecs_tasks" {
+  source = "baikonur-oss/iam-nofile/aws"
+  version = "1.0.2"
+  type = "ecs-tasks"
+  name = "${var.env}${var.env_num}-${var.role}"
```
Comment (toVersus) [inline README.md]:
```suggestion
  name = "dev-batch"
```

**Review by toVersus — COMMENTED** [review]
LGTM

**Pull request merged**